### DISCUSSION TITLE
Laser Cutting Delay To Full Power

### DISCUSSION THREAD

**Discussion by outlawjack99** [arkypita/LaserGRBL#957] [Q&A]
Hi All,

My laser has a delay when trying to cut an image where it starts to cut around 9 seconds after the motion has started and it doesn't complete the sections of the image. I'm using a three part M character for testing and it misses the start and doesn't complete a complete section. I don't know if this is to do with the settings I use when "import raster image", on the main screen or something deeper.

![m](https://user-images.githubusercontent.com/63794364/79480984-cdda0d00-8006-11ea-8c1b-1ab90e684082.jpg)

**Comment by arkypita** [comment]
9 seconds? its a lot of time!

Does the laser not turn on for the first 9 seconds, or does it turn on but doesn't cut? What material do you want to cut? What colour?

**Comment by outlawjack99** [comment]
It seems a bit random in its choices. The laser always comes on, starts moving then it goes sharper and starts to cut. The new pic shows the order in which it cut each section and for 1 it started immediately but didn't finish, 2 it went through most of the motion before it started cutting and 3 was more or less the same. I don't think the imported image is at fault unless I'm setting the image up wrong?

Curiously it cut all but the very last bit of the other pic using the exact same settings.

![WIN_20200417_10_08_54_Pro](https://user-images.githubusercontent.com/63794364/79552957-ae3cf600-8093-11ea-9970-94f5a7687f12.jpg)


![WIN_20200417_09_59_21_Pro](https://user-images.githubusercontent.com/63794364/79552130-839e6d80-8092-11ea-9ebe-1707db692205.jpg)

**Comment by gmmanonymus111** [comment]
While cutting white paper, you need to delay your movement a bit for the laser to actually start cutting. Once it burned the paper, it can continue & finish cutting. White paper reflect laser light, and caused similar issues with my 1.5w laser.

**Comment by outlawjack99** [comment]
Well I'll be. I tried it on some brown then red paper and it worked a treat. So I take it the white stops the laser from "biting"? Either way it seems a thank you is in order.

**Comment by gmmanonymus111** [comment]
> So I take it the white stops the laser from "biting"?

Exactly.
I'm glad it worked out.
